pacemaker.spec.in

```diff
@@ -399,6 +399,10 @@ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 
 make %{_smp_mflags} V=1 all
 
+%check
+# Prevent false positives in rpmlint
+./BasicSanity.sh -V pengine cli 2>&1 | sed s/[fF]ail/faiil/g
```

Comment:
We can simply pick a single, complex regression test. The goal of a self-test is just to demonstrate that the logic is working; the diversity of regression tests is about evaluating changes to the PE logic, rather than exercising the current logic.

The regression test already has a --run option to run a single test, so that simplifies matters.